tests/test_rocksdb_options.rs

```diff
@@ -132,3 +132,16 @@ fn test_set_wal_opt() {
     drop(db);
 }
 
+#[test]
+fn test_create_info_log() {
+    let path = TempDir::new("_rust_rocksdb_test_create_info_log_opt").expect("");
+    let mut opts = Options::new();
```

Comment:
Do we need set `log_file_time_to_roll` `max_log_file_size` and `db_log_dir` for option?